android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -116,13 +134,24 @@ class RNPlayerView(
         }
 
         override fun onStop(owner: LifecycleOwner) {
+            if (player?.config?.lockScreenControlConfig?.isEnabled == false) {
+                mediaSessionServicePlayer = null
+            } else {
+                player = null
+            }
+
             playerView?.onStop()
         }
 
         override fun onDestroy(owner: LifecycleOwner) = dispose()
     }
 
     init {
+        playerEventRelay = EventRelay<Player, Event>(
+            EVENT_CLASS_TO_REACT_NATIVE_NAME_MAPPING,
+            ::emitEventFromPlayer,
+        )
+
```

Comment:
I had to move `playerEventRelay` initialization here because it was `null` during the first invocations of `onStop`.
This is due to the custom `setPlayer` we have, which also sets `playerEventRelay`